example/build.gradle

```diff
@@ -47,4 +48,25 @@ dependencies {
     reactor_3_3_x 'io.projectreactor:reactor-core:3.3.0.BUILD-SNAPSHOT', {
         changing = true
     }
+
+    jmh project(path: ":agent", configuration: 'shadow')
+    //Uncomment for benchmark execution by Idea plugin
+//    jmh "org.openjdk.jmh:jmh-generator-annprocess:1.21"
```

Comment:
I guess we can keep it uncommented, it won't hurt, but less manual actions to run it from IDEA after a fresh checkout

Reply:
Unfortunately not. The plugin adds it itself and there is an error about duplicated generated classes. There is a possibility of changing `duplicateClassesStrategy`, but it can have side effects.
It would be best to detect import from Idea and only then add that dependency, but I haven't found any good way to do that.